### DISCUSSION TITLE
(osu!mania) Add back the key count next to the difficulty to match stable.

### DISCUSSION THREAD

**Discussion by Terminxa** [ppy/osu#20727] [Ideas] [answered]
Here is how it looks on osu!lazer:
![image](https://user-images.githubusercontent.com/106455196/195447530-7b0d92f8-8e2c-4ff4-8547-fba160b2251a.png)
Here is how it looks on stable:
![image](https://user-images.githubusercontent.com/106455196/195447559-74d7f1be-7b72-4a97-ad81-1cb463e62ea0.png)

**Comment by frenzibyte** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/9453 / https://github.com/ppy/osu/issues/5668.
